elixir/native/extism_nif/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+use rustler::{Error, Atom};
+use extism::{Plugin};
+use std::str;
+use serde_json;
+use std::path::Path;
+use std::str::FromStr;
+
+mod atoms {
+    rustler::atoms! {
+        ok,
+        error,
+        eof,
+        unknown // Other error
+    }
+}
+
+// fn load(env: Env, _: Term) -> bool {
+//     true
+// }
+
+ #[rustler::nif]
+fn plugin_new_with_manifest(manifest_payload: String, wasi: bool) -> Result<isize, Error> {
+    match serde_json::from_str(&manifest_payload) {
+        Ok(manifest) => {
+            match Plugin::new_with_manifest(&manifest, wasi) {
+                Err(_e) => Err(Error::Term(Box::new("Could not load plugin"))),
```

Comment:
I'm going to come back with some better error handling soon. This is just for development purposes.